Files/UserControls/MultitaskingControl/HorizontalMultitaskingControl.xaml

```diff
@@ -25,15 +25,15 @@
                         Opening="TabItemContextMenu_Opening">
                         <MenuFlyoutItem
                             x:Uid="HorizontalMultitaskingControlNewTab"
-                            Click="{x:Bind views:MainPage.AddNewTabAtIndex}"
+                            Click="{x:Bind vm:MainPageViewModel.AddNewTabAtIndex}"
```

Comment:
It doesn't need to be included in this PR, but we should remove the reference to MainPageViewModel and implement proper dependency properties so that the control is completely decoupled.

Reply:
@yaichenbaum :+1: The function I'm referencing there is static anyway.

Reply:
@d2dyno1 It's fine for now since it was already like this, I opened an issue to track this. #4221